src/sparseml/pytorch/utils/mfac_helpers.py

```diff
@@ -17,6 +17,7 @@
 pruning.
```

Comment:
as we're getting more logic changes within the HInv classes, it would be great for us to add some testing that would compare results of HInv calcs across the implementations for the same matrix. feel free to sync on this if needed